version.config

```diff
@@ -1,3 +1,3 @@
 version = 2.6.0-dev
-update_from_version = 2.5.2
+update_from_version = 2.6.0
```

Comment:
Shouldn't this also bump `2.6.0-dev` to `2.7.0-dev` in master since it is a minor release?

Reply:
I'm not sure about this, I see that for 2.5 it was done in the post-release PR. I got some obscure errors when I tried to bump it right away. Reference:

2.5 release: https://github.com/timescale/timescaledb/pull/3744/files#diff-4f27bf8a71d11784a71b440d480d2b4d6350380dac8bd8d13807746fed15ffda

2.5 post-release: https://github.com/timescale/timescaledb/commit/df7f058ad13802a34290ebe332962b95296568c6#diff-4f27bf8a71d11784a71b440d480d2b4d6350380dac8bd8d13807746fed15ffda

Reply:
I think this is material for the Post Release PR, no?

Reply:
I did it in the post release PR when I worked in the 2.5.0: https://github.com/timescale/timescaledb/pull/3751/files#diff-4f27bf8a71d11784a71b440d480d2b4d6350380dac8bd8d13807746fed15ffda